ui/apps/platform/src/Containers/Search/searchCategories.ts

```diff
@@ -119,15 +108,12 @@ function getSearchResultCategoryMap(
             filterOn: null,
             viewLinks: [
                 {
-                    basePath: isVm2Ga
-                        ? `${vulnerabilityNamespaceViewPath}${getQueryString({
-                              // TODO - Add regex searching support for namespace view ROX-24484 when ROX_VULN_MGMT_2_GA is enabled
-                              s: {
-                                  NAMESPACE: [':name'],
-                                  CLUSTER: [':locationTextForCategory'],
-                              },
-                          })}`
-                        : `${vulnManagementNamespacesPath}/:id`,
+                    basePath: `${vulnerabilityNamespaceViewPath}${getQueryString({
+                        s: {
+                            Namespace: ['^:name$'],
+                            Cluster: ['^:locationTextForCategory$'],
+                        },
+                    })}`,
```

Comment:
This adds support for _exact_ namespace/cluster matches via regex as well as fixes the casing of the search parameters. On staging, you can see that visiting VM from a search page namespace link causes the search chips to not be visible.